CHANGELOG.md

```diff
@@ -1,12 +1,18 @@
 # CHANGELOG
 
+## 2.0.0-rc1
```

Comment:
ouch, I caused some confusion here, the version 2.0.0 was already published and tagged in this repository. I should have put a separate entry in the changelog to make it clear, but since there were no changes comparing to `beta.3` I forgot doing so. I will cleanup the situation.